.ci.yaml

```diff
@@ -4323,3 +4323,14 @@ targets:
     properties:
       tags: >
           ["framework","hostonly","shard"]
+
+  - name: Mac_android android_choreographer_do_frame_test
```

Comment:
An entry is needed in [TESTOWNERS](https://github.com/flutter/flutter/blob/master/TESTOWNERS) to unblock the `Linux test_ownership` failure.

Reply:
Done